### PR TITLE
Makes symptoms a bit more stealthy, faster spreading and slightly increases killing potential

### PR DESCRIPTION
**Pull request by ma44**
As title says (no lynch please)

### CODE REVIEW

**Comment by xxalpha** [comment]
please squash

**Comment by GunHog** [comment]
I know I might be banned for saying this, but you really need to consider compiling your code and making a few viruses to ensure things are working how you intended. Also, make your changes locally on your computer, not with the editor - that will allow you to make things in one commit.

Friendly warning to you. **BACKUP YOUR FILES BEFORE YOU TRY TO "squash" FOR THE FIRST TIME. IF YOU DO NOT KNOW WHAT YOU ARE DOING, YOU WILL DESTROY YOUR BRANCH.**

**Comment by Bawhoppen** [comment]
@xxalpha don't maintainers have a button to auto-squash when they merge

**Comment by optimumtact** [comment]
Don’t' include the old stats in the comments. Git is responsible for keeping historical info. someone can always check out the commit just before the one that merges your change to review the old data.

**Comment by XDTM** [comment]
Balance-wise this will really hurt healing viruses since lowering stealth will be harder. You'll get stealth 3 way too easily, and then you can't read your virus anymore.

**Comment by AlexCrimson** [comment]
Have you tested this at all? Looking at the numbers, you have buffed pretty much every symptom that is used in the making of deadly diseases. Making a super stealthy AND resistant disease seems incredibly overpowered. The numbers are meant to be balanced in a way that makes you choose one or another.

And as @XDTM says its really annoying to make a good virus when you cannot see it in the Pandemic Machine.

**Comment by ma44** [comment]
In response to the other comments
@XDTM an easy solution to that would be making pandemic able to see viruses with higher stealth (which i should buff by 1-2) or even just make it detect any viruses
@AlexCrimson The deadly traits (for an example, flesh eating) the damage done with flesh eating is reduced by the amount of stealth, so the only thing that has been buffed with them is fast stage speed and transmission. This also applies to the spontaneous combustion

**Comment by GunHog** [comment]
Please include the PANDEMIC change in this PR itself, because without it, you are hurting non-antag virology players. 

You really do need to try these changes out locally, only then will you understand the suffering caused by trying to work with certain symptoms.

**Comment by ma44** [comment]
Added in PANDEMIC tweaks, makes it so health scanners can slightly see if a person has a virus, and the pandemic machine can easily see a virus.

File: code/datums/diseases/advance/symptoms/beard.dm [hunk 1]
```diff
@@ -8,6 +8,15 @@ Facial Hypertrichosis
 	Reduced transmittability
 	Intense Level.
 
+Old stats
```
Comment (optimumtact) [inline code/datums/diseases/advance/symptoms/beard.dm]: GET THIS OUTTA HERE

**Comment by AlexCrimson** [comment]
Wouldnt making the Scanner and Pandemic machine able to see viruses.... defeat the point of Stealth?

**Comment by Shadowlight213** [comment]
>buffing the positive heal viruses
Fuck no.

**Comment by optimumtact** [comment]
This feels to me like a roughshod fix and I don't think you've really justified why.

If virology is in need of so much change then it needs more than just the symptoms tweaked. It would need a fundamental rethink about how it works.

**Pull request closed without merging**